src/pybind/mgr/orchestrator/module.py

```diff
@@ -706,15 +706,12 @@ def _apply_osd(self,
                    format: Format = Format.plain,
                    unmanaged: Optional[bool] = None,
                    dry_run: bool = False,
-                   no_overwrite: bool = False,
-                   inbuf: Optional[str] = None) -> HandleCommandResult:
+                   no_overwrite: bool = False) -> HandleCommandResult:
         """
         Create OSD daemon(s) using a drive group spec
         """
```

Comment:
This description doesn't feel very good anymore if we can only use --all-available-devices here. Maybe we should just have the description say this command is used for putting an osd on all available devices since that will now be it's only allowed use case (possibly with mention of unmanaged since that's the only other option to set).